src/payment/strategies/digitalriver/digitalriver-payment-strategy.spec.ts

```diff
@@ -348,25 +354,39 @@ describe('DigitalRiverPaymentStrategy', () => {
                 {
                     methodId: 'authorizenet',
                     paymentData: {
-                        nonce: JSON.stringify({
-                            checkoutId: '12345676543',
-                            source: {
-                                source: {
-                                    id: '1',
-                                    reusable: false,
-                                },
-                                readyForStorage: true,
+                        formattedPayload: {
+                            credit_card_token: {
+                                token: JSON.stringify({
+                                    checkoutId: '12345676543',
+                                    source: {
+                                        source: {
+                                            id: '1',
+                                            reusable: false,
+                                        },
+                                        readyForStorage: true,
+                                    },
+                                    sessionId: '1234',
+                                }),
                             },
-                            sessionId: '1234',
-                        }),
+                            set_as_default_stored_instrument: false,
+                            vault_payment_instrument: true,
+                        },
                     },
                 }
             );
         });
 
+        it('throws an error when DigitalRiver checkout data is not provided', async () => {
```

Comment:
This test seems to be not correct, the Digital River Checkout data isn't into the payload.

Reply:
Sorry, I confused the name of the test with the one I have above, let me fix it